stanford/playbooks/forum.yml

```diff
@@ -14,8 +14,6 @@
     - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
     - "{{ COMMON_DEPLOYMENT_DIR }}/{{ CLUSTER_NAME }}.yml"
   roles:
-    - oraclejdk
-    - elasticsearch
```

Comment:
We no longer need ES directly on the forum box,
since we're providing a dedicated cluster.